spec/03-plugins/23-rate-limiting/04-access_spec.lua

```diff
@@ -100,1301 +100,1332 @@ local redis_confs = {
 
 
 for _, strategy in helpers.each_strategy() do
-  for _, policy in ipairs({ "local", "cluster", "redis" }) do
-      for redis_conf_name, redis_conf in pairs(redis_confs) do
-        if redis_conf_name ~= "no_ssl" and policy ~= "redis" then
-          goto continue
-        end
+for _, policy in ipairs({ "local", "cluster", "redis" }) do
+for redis_conf_name, redis_conf in pairs(redis_confs) do
+for _, sync_rate in ipairs({0, 0.01}) do
```

Comment:
I suggest adding a dedicated test for `sync_rate`. It should include 2 cases:
1. After sync, the limit is exceeded;
2. Before sync, the local limit is exceeded.